network-store-client/src/main/java/com/powsybl/network/store/client/RestClientImpl.java

```diff
@@ -45,9 +48,10 @@ public static RestTemplateBuilder createRestTemplateBuilder(String baseUri) {
     }
 
     private static ObjectMapper createObjectMapper() {
-        ObjectMapper objectMapper = new ObjectMapper();
-        objectMapper.registerModule(new JodaModule());
-        return objectMapper;
+        return JsonUtil.createObjectMapper()
```

Comment:
```suggestion
        return new ObjectMapper()
```
I suggest to stay in line with the original implementation using `ObjectMapper()`, this change leads to issue in the network-store-server when switching to `JsonUtil.createObjectMapper()`.

Reply:
done